doc/reference_python/SWESimulators/CTCS.py

```diff
@@ -60,14 +61,24 @@ class CTCS:
     wind_stress: Wind stress parameters
     """
     def __init__(self, \
-                 cl_ctx, \
+                 #cl_ctx, \
                  H, eta0, hu0, hv0, \
                  nx, ny, \
                  dx, dy, dt, \
                  g, f, r, A, \
                  wind_stress=Common.WindStressParams(), \
                  block_width=16, block_height=16):
-        self.cl_ctx = cl_ctx
+        #self.cl_ctx = cl_ctx # How could this work...? (Replaced by the ten lines below.)
```

Comment:
This class should not create its own OpenCL context. It should receive a context to run.